nbs/explains/minidataapi.ipynb

```diff
@@ -0,0 +1,516 @@
+{
```

Comment:
_Commented on notebook `nbs/explains/minidataapi.ipynb` Cell 11 Line 4_
```python
# Now the todos table
todos.insert(dict(title='Write MiniDataAPI spec', status='open', name='Braden'))
todos.insert(dict(title='Implement SSE in FastHTML', status='open', name='Alma'))
todos.insert(dict(title='Launch FastHTML', status='closed', name='Charlie'))
```
This is the main interesting one to point out -- that the returned todo has the updated id in it. It's not just the object we passed in. Also the dict has been converted to the class. These are the important details to not only show, but also explain why this is important and how it's used in fasthtml (for example).

i.e we need to explain *why* the spec has these particular details.